guides/source/active_storage_overview.md

```diff
@@ -508,9 +508,9 @@ url_for(user.avatar)
 WARNING: The links generated by ActiveStorage are hard to guess, but publicly
 accessible by default. Anyone that knows the blob URL will be able to download it,
 even if a `before_action` in your `ApplicationController` would otherwise
-require a login. If your files require a higher level of protection consider
-implementing your own authenticated
-[`ActiveStorage::Blobs::RedirectController`](https://github.com/rails/rails/blob/main/activestorage/app/controllers/active_storage/blobs/redirect_controller.rb) and [`ActiveStorage::Representations::RedirectController`](https://github.com/rails/rails/blob/main/activestorage/app/controllers/active_storage/representations/redirect_controller.rb).
+require a login. If you are going to handle sensible files, such as personal documents,
+it's highly recommended to override ActiveStorage controllers with your owns and implement
+authentication and authorization in them. The controllers are [`ActiveStorage::DirectUploadsController`](https://github.com/rails/rails/blob/main/activestorage/app/controllers/active_storage/direct_uploads_controller.rb), [`ActiveStorage::Blobs::RedirectController`](https://github.com/rails/rails/blob/main/activestorage/app/controllers/active_storage/blobs/redirect_controller.rb), [`ActiveStorage::Blobs::ProxyController`](https://github.com/rails/rails/blob/main/activestorage/app/controllers/active_storage/blobs/proxy_controller.rb), [`ActiveStorage::Representations::RedirectController`](https://github.com/rails/rails/blob/main/activestorage/app/controllers/active_storage/representations/redirect_controller.rb) and [`ActiveStorage::Representations::ProxyController`](https://github.com/rails/rails/blob/main/activestorage/app/controllers/active_storage/representations/proxy_controller.rb).
```

Comment:
Not sure if include the `ActiveStorage::DirectUploadsController` or mention it separately, because the risk is not leaking sensible data, but having our storage systems attacked